src/test_linked_list.py

```diff
@@ -0,0 +1,127 @@
+"""Testing the linked_list class - CF 401 Python Week 2 Assignment."""
+import pytest
+
+PARAMS_SAMPLE_LIST = ["something", 1, "pear", 3, "apple"]
+
+@pytest.fixture
+def init_node():
+    from linked_list import Node
+    node = Node(0)
+    return str(type(node))
+
+@pytest.fixture
+def create_empty_node():
+    from linked_list import Node
+    return Node()
+
+@pytest.fixture
+def create_ll_with_list():
+    """Given a list, create a new Linked List with all the values."""
+    from linked_list import Linked_List
+    new_ll = Linked_List(PARAMS_SAMPLE_LIST)
+    return new_ll
+
+@pytest.fixture
+def create_empty_ll():
+    from linked_list import Linked_List
+    new_ll = Linked_List()
+    return new_ll
+
+
+def test_node_init():
+    """Test the initialization of a node."""
+    from linked_list import Node
+    assert type(Node) == type
```

Comment:
All class objects are `type` objects. You don't need to test for that, that's handled by Python.